command/extract_dist.py

```diff
@@ -22,6 +22,14 @@ def __init__(self, *args, **kwargs):
                      'packages', 'py_modules', 'scripts']:
             self.metadata[attr] = to_list(getattr(self.distribution, attr, []))
 
+        try:
+            self.metadata['extras_require'] = []
+            for v in getattr(self.distribution, 'extras_require', {}).values():
+                self.metadata['extras_require'] += to_list(v)
+        except AttributeError:
```

Comment:
Should not we also catch `ValueError` here (which can be raised in `to_list`) or is it a rare case?